### DISCUSSION TITLE
Does node views work with SSR?

### DISCUSSION THREAD

**Discussion by alexbjorlig** [ueberdosis/tiptap#3662] [General Questions]
Our team is trying to build our first node view with Svelte, based on this [repo](https://github.com/sibiraj-s/svelte-tiptap).

Everything seems to work with the counter-example - but does not work with SSR.

Should SSR not be supported out of the box here?

**Comment by einarpersson** [comment]
I am also new to this, and I think SSR is not supported. Looking at [the official docs](https://tiptap.dev/docs/editor/installation/svelte#3-create-a-new-component) I see that the sample editor is instantiated in `onMount` i.e. client side. 

It would be good if this would work SSR as well, since that would both make the API a bit simpler, and also it would allow for using Tiptap to also render the not editable result to the end user. Currently we are using `generateHtml` from `@tiptap/html` but that only produces static HTML and cannot as such produce Svelte components.

What we do is that we have a {#each } where we look if the current content entry is a Component (then use `<svelte:component ...`) and if not use `{ @html generateHtml(...) `.
